robottelo/api/utils.py

```diff
@@ -130,6 +133,38 @@ def publish_puppet_module(puppet_modules, repo_url, organization_id=None):
     return cv.read()
 
 
+def import_puppet_module(sat, puppet_module_repo=CUSTOM_PUPPET_MODULE_REPOS['generic_1']):
```

Comment:
Every function that takes a satellite, capsule, or content host as its argument, I'll be asking:

Why isn't this a class method on the Satellite class?

Reply:
It certainly could be

Reply:
will move thanks